### PR TITLE
junit 5 assignment

### PR DESCRIPTION
**Pull request by jasleenwahi**
please review my junit 5 assignment

### CODE REVIEW

File: junit5assignment/src/test/java/com/knoldus/PersonTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+public class PersonTest {
+
+    private Person person;
+    @BeforeEach
+    public void setup(){
+        person = new Person();
+    }
+
+    @Test
+    public void nameTest(){
```
Comment (anup0060) [inline junit5assignment/src/test/java/com/knoldus/PersonTest.java]: not all scenarios covered 

and method naming not good - use something like - test<ActualMethodName>_given<Input> ex - testName_givenNameNull

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Review by jasleenwahi — COMMENTED** [review]
I have made the changes suggested by you, thank you for the feedback sir it made me learn and improve my code.

**Review by jasleenwahi — COMMENTED** [review]
Changes made, thank you for the review sir it helped me lot in learning.

File: junit5assignment/src/test/java/com/knoldus/PersonTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    private Person person;
+    @BeforeEach
+    public void setup(){
+        person = new Person();
+    }
+
+    @Test
+    @DisplayName("Test getName() with not-null value")
+    void testGetName_With_Non_Null_Value() {
```
Comment (anup0060) [inline junit5assignment/src/test/java/com/knoldus/PersonTest.java]: _given<>_should<>

_givenNotNull_shouldReturnNotNull

**Review by anup0060 — CHANGES_REQUESTED** [review]

**Review by jasleenwahi — COMMENTED** [review]
Made the required changes, thankyou for the review sir.

**Review by jasleenwahi — COMMENTED** [review]
Changed the convention, thank you for making me understand better naming convention

**Review by anup0060 — APPROVED** [review]

**Pull request merged**